page.css

```diff
@@ -1,5 +1,4 @@
 ---
-layout: null
```

Comment:
Works for months now.
On May 15, 2015 10:42 PM, "░▒▓█│【Walkman】│█▓▒░" <notifications@github.com>
wrote:

> In page.css
> <https://github.com/mdo/github-buttons/pull/79#discussion_r30439644>:
>
> > @@ -1,5 +1,4 @@
> >  ---
> > -layout: null
>
> Does this still work?? If so, very cool. (I thought the layout tag was
> required)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdo/github-buttons/pull/79/files#r30439644>.
>